R/date_management.R

```diff
@@ -136,3 +136,60 @@ calc_submission_due_date <- function(forecast_date, submission_day = "Monday") {
   
   return(dates)
 }
+
+#' Create a datetime by appending time and/or UTC timezone with offset 
+#' to the given date string, if necessary
+#' 
+#' If you would like to set a timezone in date parameter, it has to be UTC now. 
+#' If not, this function will append the default timezone to your input based on hub parameter. 
+#' 
+#' @param date date in character. 
+#' It could be a YYYY-MM-DD date, 
+#' or a YYYY-MM-DD date format with HH:MM:SS time, 
+#' or a YYYY-MM-DD date with HH:MM:SS time and UTC timezone.
+#' Default to NULL
+#' @param hub character vector, where the first element indicates the hub
+#' to set default timezone. Possible options are "US" and "ECDC".
+#' 
+#' @return NULL or datetime characters with date, time, UTC timezone and optional offset 
+#' 
+#' @export
+date_to_datetime <- function(date = NULL, hub = c("US", "ECDC")){
+  # default for `as_of` in zoltr
+  if(is.null(date)){
+    return (NULL)
+  }
+  
+  if (hub[1] == "US"){
+    default_timezone_offset <- "-5"
```

Comment:
Wondering if we want this function to follow daylight savings time zone adjustments...